wagtailcaptcha/forms.py

```diff
@@ -4,10 +4,12 @@
 
 
 class WagtailCaptchaFormBuilder(FormBuilder):
+    CAPTCHA_FIELD_NAME = 'wagtailcaptcha'
+
     def __init__(self, fields):
         super(WagtailCaptchaFormBuilder, self).__init__(fields)
         # Add wagtailcaptcha to FIELD_TYPES declaration
-        self.FIELD_TYPES.update({'wagtailcaptcha': self.create_wagtailcaptcha_field})
+        self.FIELD_TYPES.update({self.CAPTCHA_FIELD_NAME: self.create_wagtailcaptcha_field})
```

Comment:
Also need to do the same at line 21:
fields['wagtailcaptcha'] = ReCaptchaField(label='')
to:
fields[self.CAPTCHA_FIELD_NAME] = ReCaptchaField(label='')